docs/data/stashcache/vo-registration.md

```diff
@@ -0,0 +1,147 @@
+for the VO's jobs.
+
+For general information about StashCache, see the [overview document](/data/stashcache/overview/).
+
+For general information about VO registration in the OSG Topology,
+see the [registration document](/common/registration/#registering-virtual-organizations).
+
+Site admins should work together with VO managers in order to perform these steps.
+
+
+Definitions
+-----------
+
+- **Namespace:** a directory tree in the federation that is used to find VO data.
+- **Public data:** data that can be read by anyone.
+- **Protected data:** data that requires authentication to read.
+
+
+Requirements
+------------
+
+In order for a VO to join the StashCache Federation, the following needs to be true:
+
+- The VO must already be registered in the OSG Topology.
+
+- The VO must pick one or more "namespaces" for their data.
+  A namespace is a directory tree in the federation where VO data is found.
+
+!!! note
+    Namespaces are global across the federation, so you must work with the StashCache Operations team
+    to ensure that your VO's namespaces do not collide with those of another VO.
+
+- The VO must allow one or more StashCache origins to host their data.
+  An origin will typically be hosted on a site owned by the VO.
+  For information about setting up an origin, see the [installation document](/data/stashcache/install-origin/).
+
+- The VO must allow one or more StashCache Caches to cache their data.
+  The more places a VO's data can be cached in, the bigger the data transfer benefit for the VO.
+  The majority of caches across OSG will automatically cache all "public" VO data.
+  Caching "protected" VO data will often be done on a site owned by the VO.
+  For information about setting up a cache, see the [installation document](/data/stashcache/install-cache/).
+
+
+Choosing namespaces
+-------------------
+
+A namespace should be easy for your users to remember but not so generic that it collides with other VOs.
+We recommend using the lowercase version of your VO as the top-level directory.
+In addition, public data, if any, should be stored in a subdirectory named `PUBLIC`,
+and protected data, if any, should be stored in a subdirectory named `PROTECTED`.
+
+Putting this together, if your VO is named Astro, you should have:
+
+- `/astro/PUBLIC` for public data
+- `/astro/PROTECTED` for protected data
+
+Separating the public and protected data in separate directory trees is preferred for technical reasons.
+
+
+Registering data federation information
+---------------------------------------
+
+In order to declare your VO's role in the StashCache federation,
+you must add StashCache information to the VO's YAML file in the OSG Topology repository.
+
+The data will look like the following:
+```
+DataFederations:
+  StashCache:
+    Namespaces:
+      <Namespaces section>
+    AllowedCaches:
+      <AllowedCaches list>
+    AllowedOrigins:
+      <AllowedOrigins list>
+```
```

Comment:
I would change this to an example, rolling with the "Astro VO" example above

Reply:
Should I then get rid of the Astro example in the namespaces section?